src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from 'react';
+
+import usersFromServer from '../../api/users';
+import { getUserById } from '../../services/getUsers';
+import { getMaxNumber } from '../../services/getMaxNumber';
+import { Todo } from '../../types/todo';
+
+type Props = {
+  onSubmit: (todo: Todo) => void;
+  todos: Todo[];
+};
+
+export const TodoForm: React.FC<Props> = ({ onSubmit, todos }) => {
+  const [title, setTitle] = useState('');
+  const [hasTitleError, setHasTitleError] = useState(false);
+  const [userId, setUserId] = useState(0);
+  const [hasUserIdError, setHasUserIdError] = useState(false);
+
+  const handlTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(event.target.value);
+    setHasTitleError(false);
+  };
+
+  const handlUserIdChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
```

Comment:
[CHECKLIST - CODE STYLE] - It's recommended to use more descriptive variable names instead of one-letter names. Instead of 'event', you can use 'changeEvent' or 'selectEvent'.